docs/analytics_engine.rst

```diff
@@ -330,38 +326,35 @@ To run your own algorithms, you may trigger it in place where you defined it.
 .. code:: python
 
     import graphscope
+    from graphscope.dataset import load_p2p_network
 
-    sess = graphscope.session()
-    g = sess.g()
+    g = load_p2p_network()
 
     # load my algorithm
     my_app = SSSP_Pregel()
 
     # run my algorithm over a graph and get the result.
     # Here the `src` is correspondent to the `context.get_config(b"src")`
-    ret = my_app(g, src="0")
+    ret = my_app(g, src="6")
```

Comment:
Is this a string "6"? or a int 6.

Reply:
String is needed here.

https://github.com/alibaba/GraphScope/blob/main/analytical_engine/core/app/pregel/pregel_context.h#L65